lms/djangoapps/instructor_analytics/tests/test_basic.py

```diff
@@ -573,39 +574,87 @@ def test_invalid_module_state(self):
         self.assertEqual(len(datarows), 0)
 
     def test_problem_with_student_answer_and_answers(self):
-        self.course = get_course(CourseKey.from_string('edX/graded/2012_Fall'))
-        problem_location = Location('edX', 'graded', '2012_Fall', 'problem', 'H1P2')
+        self.course = CourseFactory.create(
+            display_name=u'test course',
+        )
+        section = ItemFactory.create(
+            parent_location=self.course.location,
+            category='chapter',
+            display_name=u'test section',
+        )
+        sub_section = ItemFactory.create(
+            parent_location=section.location,
+            category='sequential',
+            display_name=u'test subsection',
+        )
+        unit = ItemFactory.create(
+            parent_location=sub_section.location,
+            category="vertical",
+            metadata={'graded': True, 'format': 'Homework'},
+            display_name=u'test unit',
+        )
+        problem = ItemFactory.create(
+            parent_location=unit.location,
+            category='problem',
+            display_name=u'test problem',
+        )
+        submit_and_compare1 = ItemFactory.create(
+            parent_location=unit.location,
+            category='submit-and-compare',
+            display_name=u'test submit_and_compare1',
+        )
+        submit_and_compare2 = ItemFactory.create(
```

Comment:
per F2F it NEEDS to be two different problem definitions, so I suggested giving them meaningful names.